docs/content/stable/develop/quality-of-service/write-heavy-workloads.md

```diff
@@ -23,66 +24,59 @@ The idea is to slow down incoming writes to the speed that the database can hand
 
+  * By setting `default_memory_limit_to_ram_ratio` to control what percentage of total RAM on the instance the process should use
+  * Specify an absolute value using `memory_limit_hard_bytes`. For example, to give YB-TServer 32GB of RAM, use `--memory_limit_hard_bytes 34359738368`
 
 * **Too many memstores queued for flush:** More than one memstore is queued for getting flushed to disk. The number of memstores queued at which this trigger is activated is set to 2 (therefore, 2 or more memstores are queued for flush). Note that in practice, there is always an active memstore, which is not included in this limit.
 
   {{< warning title="Warning" >}}
   The trigger for too many memstores queued for flush is controlled by the flag `max_write_buffer_number`, which defaults to 2. However, it is not recommended at this time to change this value.
   {{< /warning >}}
 
-
 ### Slowdown writes trigger
+
 Slowdown writes trigger is activated when the number of SST files exceeds the value determined by the flag `sst_files_soft_limit`, but does not exceed the `sst_files_hard_limit` value. The default value of `sst_files_soft_limit` flag is 24. The slowdown in writes is achieved by rejecting a percentage of incoming writes with probability `X`, where `X` is computed as follows:
 
 ```
-    X = (<num SST files> - soft_limit) / (hard_limit - soft_limit)
+X = (<num SST files> - soft_limit) / (hard_limit - soft_limit)
 ```
 
 ## Admission control trigger and enforcement
 
-The incoming write rejection is computed at a per *tablet* basis. Recall that a tablet consists of tablet peers that participate in Raft consensus, and has a tablet leader elected. The rejection is triggered if one of the conditions described in the prior section occurring on the tablet leader OR a majority of the tablet peers. The idea here being that, if a single follower lags behind, it is sufficient to just let it catch up later. However, if either the leader or a majority of the followers is slow, this might reflect as latency to the request anyway and therefore warrants some admission control.
+The incoming write rejection is computed at a per *tablet* basis. Recall that a tablet consists of tablet peers that participate in Raft consensus, and has a tablet leader elected. The rejection is triggered if one of the conditions described in the prior section occurring on the tablet leader OR a majority of the tablet peers. The idea here being that, if a single follower lags behind, it is sufficient to just let it catch up later. However, if either the leader or a majority of the followers is slow, this might reflect as latency to the request anyway, and therefore warrants some admission control.
```

Comment:
The incoming write rejection is computed per tablet. Recall that a tablet consists of tablet peers that participate in Raft consensus, and has a tablet leader elected. The rejection is triggered if one of the conditions described in the previous section occurring on the tablet leader or a majority of the tablet peers. That is, if a single follower lags behind, it is sufficient to let it catch up later. However, if either the leader or a majority of the followers is slow, this might reflect as latency to the request, and therefore warrants some admission control.